src/mysky.ts

```diff
@@ -344,20 +351,20 @@ export class MySky {
       // Launch the new permissions provider.
       this.permissionsProvider = launchPermissionsProvider(seed);
 
-      // If JWT is found, then set it on the client.
-      const jwt = localStorage.getItem(JWT_STORAGE_KEY);
-      if (jwt) {
-        // Clear the stored JWT.
+      // If the email is found, then set up auto-login on Main MySky.
+      const email = localStorage.getItem(EMAIL_STORAGE_KEY);
+      if (email) {
+        // Clear the stored email.
         //
-        // The JWT can be cleared here because `localStorage` is only used to
-        // marshal the JWT from MySky UI over to the invisible MySky iframe. We
-        // don't clear the seed because we need it in storage so that users are
-        // automatically logged-in, when possible. But for the JWT, it should be
-        // stored on MySky, as the local storage can get cleared, users can move
-        // across browsers etc.
-        localStorage.removeItem(JWT_STORAGE_KEY);
-
-        this.client.customOptions.customCookie = `skynet-jwt=${jwt}`;
+        // The email can be cleared here because `localStorage` is only used to
+        // marshal the email from MySky UI over to the invisible MySky iframe.
+        // We don't clear the seed because we need it in storage so that users
+        // are automatically logged-in, when possible. But for the email, it
+        // should be stored on MySky, as the local storage can get cleared,
+        // users can move across browsers etc.
+        localStorage.removeItem(EMAIL_STORAGE_KEY);
+
+        // TODO: Set up auto-login.
```

Comment:
Yes, I've already handled this TODO in the upcoming PR. 😉

>  Should we remove the stored email after we've handled it?

I think it's better to remove it before, in case there is an error in handling it.